web/templates/admin/attendee/index.html.eex

```diff
@@ -0,0 +1,27 @@
+<h2>Listing attendees</h2>
+
+<table class="table">
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Email</th>
+
+      <th></th>
+    </tr>
+  </thead>
+  <tbody>
+<%= for attendee <- @attendees do %>
+    <tr>
+      <td><%= attendee.name %></td>
+      <td><%= attendee.email %></td>
+
+      <td class="text-right">
+        <%= link "Show", to: admin_attendee_path(@conn, :show, attendee), class: "btn btn-default btn-xs" %>
+        <%= link "Edit", to: admin_attendee_path(@conn, :edit, attendee), class: "btn btn-default btn-xs" %>
+        <%= link "Delete", to: admin_attendee_path(@conn, :delete, attendee), method: :delete, data: [confirm: "Are you sure?"], class: "btn btn-danger btn-xs" %>
```

Comment:
Minor: I'd wrap up this pattern into a delete_link since you've already used it twice in this PR (and likely to use it elsewhere)

```elixir
delete_link("Delete", admin_attende_path(@conn, :delete, attendee))
```